lib/caps/ICL/iHD

```diff
@@ -0,0 +1,77 @@
+###
+### Copyright (C) 2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+caps = dict(
+  decode  = dict(
+    jpeg    = dict(maxres = (8192, 8192)),
+    mpeg2   = dict(maxres = (4096, 4096)),
+    vc1     = dict(maxres = (4096, 4096)),
+    avc     = dict(maxres = (4096, 4096)),
+    hevc_8  = dict(maxres = (8192, 8192)),
+    hevc_10 = dict(maxres = (4096, 4096)),
+    vp8     = dict(maxres = (4096, 4096)),
+    vp9_8   = dict(maxres = (4096, 4096)),
+    vp9_10  = dict(maxres = (4096, 4096)),
```

Comment:
AFAIK, maxres should be (8192,8192) for vp9 too on ICL

Reply:
For both 8 and 10 bit? Decode and vdenc?

Reply:
hevc and vp9 use the same HCP unit, so the maximum resolution should be same for hevc and vp9. I am not sure whether 10bit and vdenc support 8K and will double check the media driver.

Reply:
Aha, just found this https://github.com/intel/media-driver/blob/master/docs/media_features.md which should answer the question 😃